src/etc/inc/interfaces.inc

```diff
@@ -902,7 +903,7 @@ function interfaces_configure($verbose = false)
         } elseif (!empty($config['interfaces'][$if]['ipaddrv6']) && $config['interfaces'][$if]['ipaddrv6'] == 'track6') {
             $track6_list[$if] = $ifname;
         } else {
-            interface_configure($verbose, $if);
+            $standard_list[$if] = $ifname;
```

Comment:
looks like the same as adding them to $bridge_list, but I'm quite sure this will cause issues for others from the looks of the amount of reprioritisation done here. no better idea yet, but thanks for looking into it